src/views/annual-report/2020/l10n.json

```diff
@@ -0,0 +1,379 @@
+    "annualReport.2020.communityQuoteText": "[In 2020], there were not many opportunities to engage with parents in such a fun, high energy way. So this opportunity provided much needed engagement...Teachers were apprehensive, but the students' level of excitement pushed them into a space where they had to trust the process and allow kids to learn from one another.",
+
+    "annualReport.2020.communityScratchCommunity": "Scratch Community",
+    "annualReport.2020.communityScratchCommunityIntro": "When asked why they use Scratch, most Scratchers talk about the importance of the online community for motivating their ongoing participation, providing a space where they can express their creativity, make friends, receive feedback, get new ideas, and learn new skills. Many Scratchers express their appreciation for the Scratch community as a safe and welcoming space to connect, share, and learn from one another.",
+
+    "annualReport.2020.communityQuoteGroupName1": "angelical",
+    "annualReport.2020.communityQuoteGroupText1": "I joined Scratch when I was 11 years old and the things I learned from using the platform and interacting with the community were really a vital part of my learning growing up.",
+    "annualReport.2020.communityQuoteGroupName2": "dlore2009",
+    "annualReport.2020.communityQuoteGroupText2": "Scratch has allowed me to do things from home, like\n- Respect people and their projects\n- Make friends\n- Feel that I am not alone in this quarantine\n....and much more, so I want to say ¡GRACIAS!",
+    "annualReport.2020.communityQuoteGroupName3": "qood",
+    "annualReport.2020.communityQuoteGroupText3": "I've been on Scratch for about 2 years, and it's been a life-changing experience! I've learned so many new things, such as coding, online etiquette, and art!",
+    "annualReport.2020.communityQuoteGroupName4": "Mechanical_pencil",
+    "annualReport.2020.communityQuoteGroupText4": "Scratch was my favorite hobby in sixth grade. It secretly introduced me to Boolean logic, order of operations, and nested mathematical expressions—not to mention computer programming itself.",
+
+    "annualReport.2020.yearInReview": "Year in Review",
+    "annualReport.2020.yearInReviewText": "2020 was a remarkable year in the online community. The Community Team highlighted and developed opportunities for young people to express their ideas and become engaged in positive ways, and incredible movements sprung up from Scratchers themselves. Here’s a look back at some of the highlights of the year:",
+
+    "annualReport.2020.yearInReviewCard1Date": "January",
+    "annualReport.2020.yearInReviewCard1Title": "End of the Decade Scratch Design Studio",
+    "annualReport.2020.yearInReviewCard1Text": "Scratchers celebrated the close of a decade and new beginnings in this Scratch Design Studio.",
+    "annualReport.2020.yearInReviewCard1Link": "https://scratch.mit.edu/studios/25528144",
```

Comment:
These links should be in the source code as opposed to this file. Or we can put them into a separate json file if that's easier. This file is the set of strings that go off to our volunteer translator community.